csuntechlab/iSTART
# STR-21 -- FE: Create routing for user to the their splash page based on their role

# Description
Routed page to dashboard after login; created logic for specified group.
  
# Testing
1. Log in with steve@csun.edu; no password. 
`NOTE: You won't be able to login if the user is not in a proper group.`
2. Ensure no errors appear.
3. Run yarn jest-unit; ensure all tests pass.
